rfcs/0031-devices-api.md

```diff
@@ -306,6 +293,8 @@ Another route to take is to treat RTOSes as entirely separate from TVM, requirin
 # Unresolved questions
 [unresolved-questions]: #unresolved-questions
 
+Is coupling `Target`s with this Device API to the C Runtime acceptable? From the authors point of view this seems an acceptable trade off given the devices won't function correctly without the flow control.
```

Comment:
i think it's likely that [TargetDevice](https://github.com/apache/tvm/pull/8892) will play a role here.

Reply:
Is there a `TargetDevice` RFC I can reference? Otherwise I'd assume `TargetDevice` is speculative rather than something we should include here?

Reply:
sorry not yet :/. it's sort of teased in the future work section of my [Artifact pre-RFC](https://discuss.tvm.apache.org/t/introduce-artifact-a-container-for-generated-code/11099). i'm more including this pointer here as an "early FYI" how i'd like to solve this (e.g. using the TargetDevice name as the mechanism to bind a sub-target to a Device API impl).